pkg/sfu/downtrack.go

```diff
@@ -265,23 +270,22 @@ func NewDownTrack(
 		kind = webrtc.RTPCodecType(0)
 	}
 
+	if _, err := params.PlayoutDelayLimit.Marshal(); err != nil {
```

Comment:
should it log and ignore instead here? if somehow there's a invalid value, it seems this would be problematic to kill the downtrack over it.

Reply:
Want to avoid an invalid value to make the subscriber can't get the correct playout delay. Will move the check to the position where config is loaded.